IrConsoleApp/Program.cs

```diff
@@ -3,15 +3,149 @@
 
```

Comment:
Please remove the directive "using System" because it's not necessary here (not necessary directives look 'grey' in VS). Also, delete empty rows 2 and 3 (usually, between directives and namespace should be only 1 empty row, but as you need to delete the directive, there should be no empty rows before namespace)